cmd/minikube/cmd/start.go

```diff
@@ -194,8 +196,26 @@ func runStart(cmd *cobra.Command, args []string) {
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
 
+	fmt.Printf("Setting up hostmount on %s:%s...\n", constants.DefaultMountDir,
+		constants.DefaultMountEndpoint)
+	// start 9p server mount
+	if !viper.GetBool(noMount) || cfg.GetMachineName() != constants.DefaultMachineName {
```

Comment:
Maybe we disable it by default to start off with?  

This is both adding the daemon to minikube mount and turning on mount by default.  Maybe split those into two PRs?  We should be able to test minikube mount daemon without turning it on by default right (with the mount integration test?)